src/line_image_to_text/filter_images_and_create_csv.py

```diff
@@ -0,0 +1,60 @@
+from pathlib import Path
+import csv
+from utils import get_image_name, add_row_to_csv
+
+
+def get_csv_data(batch_id):
+    with open(f'./data/{batch_id}_DB.csv', newline='') as csvfile:
+        csv_list = list(csv.reader(csvfile, delimiter=','))
+        image_list = []
+        for row in csv_list[1:]:
+            image_name = row[0]
+            image_list.append(image_name)
+    return image_list
+
+def get_images(image_keys):
+    image_names = []
+    for image_key in image_keys[1:]:
+        image_name = image_key.split("/")[-1]
+        image_names.append(image_name)
+    return image_names
+
+
+def filter_images(prefix, batch_id):
+    remaining_images = []
+    image_keys = get_image_name(prefix)
+    image_names = get_images(image_keys)
+    db_data = get_csv_data(batch_id)
+    for image_name in image_names:
+        if image_name in db_data:
+            continue
+        else:
+            remaining_images.append(image_name)
+    return remaining_images
+
+
+
+def get_csv(image_list, batch_id):
+    csv_path = Path(f"./{batch_id}.csv")
+    headers = ["id","group_id","batch_id","state","inference_transcript","url"]
+    add_row_to_csv(headers, csv_path)
+    for image_name in image_list:
+        id = image_name
+        group_id = "12"
```

Comment:
if you could put neccessary args this way, it would be more convenient when working with other group_ids,
same for other args as well.
if this wont change at all, then import from config file

def get_csv(image_list, batch_id, group_id:str="12"):
    ......